frontend/src/routes/+page.svelte

```diff
@@ -146,7 +147,7 @@
 		{/if}
 	</div>
 </div>
-<div class="w-[70%] mx-auto divider">Or select Knowledge Graph from the list below</div>
+<div class="w-[70%] mx-auto divider mt-[2rem]">Or select Knowledge Graph from the list below</div>
```

Comment:
Just a minor thing, "Or select Knowledge Graph from the list below" should be "Or select **a** Knowledge Graph from the list below"

Reply:
fixed